test/integration/apis/btc-core.test.ts

```diff
@@ -5,6 +5,12 @@ import { createPolkadotAPI } from "../../../src/factory";
 import { defaultEndpoint } from "../../config";
 import { BitcoinCoreClient } from "../../utils/bitcoin-core-client";
 
+// Test files are run alphabetically
+// but we want the oracle test to be run first
+// because it checks for the exchange rate, which
+// other tests later change
+require("./oracle.test.ts");
```

Comment:
This makes testing in isolation tricky, I would probably prefer that tests don't require an explicit ordering to pass.